### PR TITLE
fix: correct three phases hydro. equilibrium with no capillarity.

### PR DESCRIPTION
**Pull request by OmarDuran**
This PR improves the logic  for the hydrostatic equilibrium with no capillarity. For the condition without a capillarity model, there are no $P_c$ curves to invert, and the compositions from the user's elevation tables are preserved. 

An integrated test is included to verify that hydrostatic equilibrium is a solution of the PDE under compatible boundary conditions without sources or wells. The test asserts that the computed equilibrium values remain unchanged throughout the simulation.

### CODE REVIEW

File: src/coreComponents/integrationTests/fluidFlowTests/testFixedDimensionalHydrostaticEquilibriumFixture.hpp [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+          subRegion.getField< fields::flow::compAmount_n >();
+
+        integer const numComp = compAmount.size( 1 );
+
+        for( localIndex k = 0; k < subRegion.size(); ++k )
+        {
+          for( integer ic = 0; ic < numComp; ++ic )
+          {
+            real64 const accumulation = std::fabs( compAmount[k][ic] - compAmount_n[k][ic] );
+            EXPECT_NEAR( accumulation, 0.0, accumulation_tolerance )
```
Comment (jafranc) [inline src/coreComponents/integrationTests/fluidFlowTests/testFixedDimensionalHydrostaticEquilibriumFixture.hpp]:
```suggestion
            real64 const accumulation = compAmount[k][ic] - compAmount_n[k][ic];
            EXPECT_NEAR( accumulation, 0.0, accumulation_tolerance )
```
I think `EXPECT_NEAR` is handling the absolute valure logic

**Review by rrsettgast — APPROVED** [review]
